guides/common/modules/proc_retrieving-metrics-in-the-web-ui.adoc

```diff
@@ -1,16 +1,13 @@
 [id='retrieving-metrics-in-the-web-ui_{context}']
 = Retrieving Metrics in the Web UI
 
-The Grafana web application provides a dashboard-style view, with default widgets displaying the values of metrics.
+You can view the Grafana dashboard, which provides default widgets displaying the values of metrics.
```

Comment:
```suggestion
You can view the Grafana dashboard which provides default widgets displaying the values of metrics.
```

Reply:
A comma before "which" should be correct, because it introduces a nonrestrictive clause.
https://www.merriam-webster.com/grammar/when-to-use-that-and-which

Reply:
I see :+1: